src/app/GitUI/CommandsDialogs/RevisionDiffControl.Designer.cs

```diff
@@ -440,13 +442,21 @@ private void InitializeComponent()
             findInDiffToolStripMenuItem.Text = "&Find file...";
             findInDiffToolStripMenuItem.Click += findInDiffToolStripMenuItem_Click;
             // 
-            // showSearchCommitToolStripMenuItem
+            // showSearchCommitGitGrepDialogToolStripMenuItem
             // 
-            showSearchCommitToolStripMenuItem.Image = Properties.Images.ViewFile;
-            showSearchCommitToolStripMenuItem.Name = "showSearchCommitToolStripMenuItem";
-            showSearchCommitToolStripMenuItem.Size = new Size(262, 22);
-            showSearchCommitToolStripMenuItem.Text = "Sear&ch files in commit...";
-            showSearchCommitToolStripMenuItem.Click += showSearchCommitToolStripMenuItem_Click;
+            showSearchCommitGitGrepDialogToolStripMenuItem.Image = Properties.Images.ViewFile;
+            showSearchCommitGitGrepDialogToolStripMenuItem.Name = "showSearchCommitGitGrepDialogToolStripMenuItem";
+            showSearchCommitGitGrepDialogToolStripMenuItem.Size = new Size(262, 22);
+            showSearchCommitGitGrepDialogToolStripMenuItem.Text = "Search commit files with git-grep...";
```

Comment:
I think "Search commit (git-grep)" is fine too, slightly shorter.

Reply:
In general, I completely disagree with the chosen wording.
We have an existing form and wording for searching in files:
![image](https://github.com/user-attachments/assets/509b1d16-15cf-4c1c-a36e-3cba2251e4b9)
![image](https://github.com/user-attachments/assets/2a7f3d27-40ed-4cdd-ada5-91f7712d174b)

I was aligning the wording with that to provide consistent UX. Now, we have "File file.." and "Search files..." - what's the difference? Both search content in commit files. Different algorithms, but ultimately the same operation - search in files.

Reply:
The suggested wording is generally different to what other apps use - e.g., VS:
![image](https://github.com/user-attachments/assets/83d8cc13-52f7-4b6b-88f7-eb7c96dcfe70)

Reply:
> I was aligning the wording with that to provide consistent UX. Now, we have "File file.." and "Search files..." - what's the difference? Both search content in commit files. Different algorithms, but ultimately the same operation - search in files.

Find is searching the files in the diff
Search commit is searching the files in the commit

It is really the existing Find that is different from "search in working directory" for search-all apps.
But changing functionality may confuse users. Maybe that test should be changed though.
regardless, I want commit in the label to differentiate this addition. "Find files in commit (git-grep)" is longer than "Search commit (git-grep)". (having git-grep in the label is a bonus)

Reply:
> The suggested wording is generally different to what other apps use - e.g., VS:

Valid point.

I could not recognize your intention from _your_ proposal (refer to the screenshot below) and arguments because "Find file/s" is not about the file content.
![grafik](https://github.com/user-attachments/assets/d01123b5-3021-47ce-a326-1a87c8f6342d)

So, a clearer wording needs "in" like "Find in files (git-grep)" - everywhere, i.e. also "Show 'Find in files (git-grep)' filter" and "Fi&nd what:" instead of "Search:".

current state of this PR:

![image](https://github.com/user-attachments/assets/07cab7d3-e807-497f-a112-65aed20ee45f)

"Find..." for comparison:

![grafik](https://github.com/user-attachments/assets/85ab01b7-e2f4-45cd-808d-25451d7a1c3f)

Reply:
And change "Find files..." etc to "Find in diff..."?

Reply:
> And change "Find files..." etc to "Find in diff..."?

If to be changed, then `Find file in diff...` or `Find diff file...`

Reply:
Will this work?

![image](https://github.com/user-attachments/assets/4cbd3f16-a4a9-4880-aa73-30582ebb4098)
![image](https://github.com/user-attachments/assets/755dd8de-7dd5-4b78-a010-9b6319e3ec11)

Reply:
I would like to have commit in the name
Find file is not  a fileter (it would be if in the file-tree)

So the highlighted names (similar for popup), my suggestions

Find in files for commit (git-grep) with regular expression
Find in files for commit (git-grep)...
Show 'Find in files for commit (git-grep)' box

Reply:
@gerhardol sounds good.
@mstv would it work for you?

Reply:
Yes, this will work for me.

Reply:
I pushed the changes and updated the screenshots in the first post.

I kept the text in watermark a) for consistency reasons and b) the context here is obvious - the commit - so the word is redundant.
![image](https://github.com/user-attachments/assets/9731e8b4-9c74-4619-bcc7-c38d5be998a5)

This drove the wording in the context menu, the Settings and in the Find dialog.